src/collections/_documentation/product/quotas.md

```diff
@@ -79,17 +85,17 @@ When spike protection is activated, we limit the number of events accepted in an
 
 What this means is that if you experience a spike, we will temporarily protect you, but if the increase in volume is sustained, the spike protection limit will gradually increase until Sentry finally accepts all events.
 
-### An example
-
-Suppose you start with 0 events. Your spike protection limit is 20/min. At 12am, you start receiving errors at a rate of 4000/hr (or 66/min). From 12am to 1am, Sentry will discard 2800 events and keep only 1200, and send you an email notifying you that spike protection has been activated. Between 1am and 2am, your spike protection limit will still be 20/min because the trailing average for the 24 hours ending 1 am (6 * 4000 / (24 * 60) or roughly 16/min) remains **below** the base spike protection rate of 20/min. Between 2am and 3am, the trailing average is 6 * 8,000 / (24 * 60) or roughly 33/min, so spike protection keeps only 33 of the 66 events sent per minute. By 5 am, the spike protection limit reaches 6 * 16000 / (24 * 60) or 66/min, and all events are accepted.
-
 ### How does spike protection help? {#how-does-spike-protection-help}
 
 Because Sentry bills on monthly event volume, spikes can easily consume your Sentry capacity for the rest of the month. If it really is a spike (**large** and **temporary** increase in event volume), spike protection drops the vast majority of events during the spike and conserves your capacity.
```

Comment:
what about changing this to 

>Because Sentry bills on monthly event volume, spikes can consume your Sentry capacity for the rest of the month. If it really is a spike (**large** and **temporary** increase in event volume), spike protection drops events during the spike to try and conserve your capacity. We also send an email notification when spike protection is activated.

I think the email goes to owners? might be worth confirming and saying who it goes to.

Reply:
@jessla it goes to the default owner - do we have a good customer facing way to explain this, or should I just say "owner"? 
https://github.com/getsentry/getsentry/blob/master/getsentry/models/subscription.py#L854